src/bun.js/bindings/InspectorTestReporterAgent.cpp

```
@@ -50,6 +50,26 @@ void Bun__TestReporterAgentReportTestFound(Inspector::InspectorTestReporterAgent
     agent->reportTestFound(callFrame, testId, str, type, parentId);
 }
 
+void Bun__TestReporterAgentReportTestFoundWithLocation(Inspector::InspectorTestReporterAgent* agent, int testId, BunString* name, BunTestType item_type, int parentId, BunString* sourceURL, int line)
+{
+    auto str = name->toWTFString(BunString::ZeroCopy);
+    auto sourceURLStr = sourceURL->toWTFString(BunString::ZeroCopy);
+
+    Protocol::TestReporter::TestType type;
+    switch (item_type) {
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)